modules/usage-of-latest-openshift-client.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+//
+// * cicd/jenkins/important-changes-to-openshift-jenkins-images.adoc
+:_mod-docs-content-type: PROCEDURE
+
+[id="usage-of-latest-openshift-client_{context}"]
+= Usage of latest {product-title} Client (oc) in OpenShift Jenkins images
+
+Starting with 4.12, the OpenShift Jenkins container images ship with the *latest available version* of the OpenShift client (`oc`) bundled inside the image, rather than the client version matching the cluster release.
```

Comment:
```suggestion
[role="_abstract]
If your Jenkins pipelines require a *specific* {oc-first} version for compatibility or reproducibility, you must explicitly configure the desired client version in the Jenkins pipeline DSL.

Starting with 4.12, the OpenShift Jenkins container images ship with the *latest available version* of the `oc` CLI tool bundled inside the image, rather than the client version matching the cluster release.
```